src/FsAutoComplete.Core/CompilerServiceInterface.fs

```diff
@@ -150,7 +161,7 @@ type FSharpCompilerServiceChecker() =
   let checker =
     FSharpChecker.Create(
       projectCacheSize = 200,
-      keepAllBackgroundResolutions = true,
+      keepAllBackgroundResolutions = false,
```

Comment:
It's the setting that is used by VS2017 - @dsyme comment about it: https://github.com/Microsoft/visualfsharp/pull/2124#discussion_r94185793

I'm actually not sure about it - since VS is 32 bit it has memory limit. We don't have this problem - personally I'd be OK with bit higher memory usage if it gives general performance boost. I guess it would be good if @dsyme commented about it.